### DISCUSSION TITLE
Nozzle temp too low (MPC Tune)

### DISCUSSION THREAD

**Discussion by Davercade** [mriscoc/Ender3V2S1#1155] [Q&A] [answered]
Hi All, I'm new to this firmware and think it's brilliant - so many thanks for that!  I'm sure I'm missing something simple. I have done the MPC and PID for Nozzle and bed and saved the settings - the printer runs perfectly at 200 degrees - but when I set the nozzle temp to 210 I get a 'nozzle temp too low' error and the print fails (seems to happen randomly). I see a similar issue in the bug section, but seems to me like I need to tune the MPC for a higher temperature and I don't know how to do this? Is this correct or am I missing something, thanks in advance! It's a 4.2.7 board with no mods!

**Comment by mriscoc** [comment, marked as answer]
Hi, `nozzle temp too low` error usually indicates that the heating block is being cooled by the extruder fan, please check that you have the silicone jacket in place.

**Reply by Davercade** [reply]
Thank you - of course!!
